back/engines/commercial/verification/app/services/verification/patches/permissions_service.rb

```diff
@@ -12,6 +12,23 @@ def denied_when_permitted_by_groups?(permission, user)
 
         denied_reason
       end
+
+      private
+
+      def base_requirements(permission)
+        requirements = super
+        if VerificationService.new.find_verification_group(permission.groups)
+          requirements['groups'][:special][:verification] = 'require'
```

Comment:
I don't think the "groups" key exists?

Reply:
Yes, notice this changed in the base branch from using an array key to a method variable so this won't work any more. Will refactor.